monitor.js

```diff
@@ -2,15 +2,24 @@
  * Monitor
  * Pokemon Showdown - http://pokemonshowdown.com/
  *
- * Various utility functions to make sure PS is running healthy.
+ * Various utility functions to make sure PS is running healthily.
  *
  * @license MIT license
  */
 'use strict';
 
 const FS = require('./fs');
 
+/**
+ * @class TimedCounter
```

Comment:
You don't need to use `@class` or `@extends` to document the class; it doesn't add anything.